src/hooks/useCurrencyLogoURIs.ts

```diff
@@ -46,7 +46,7 @@ function getTokenLogoURI(address: string, chainId: SupportedChainId = SupportedC
   // Celo logo is hosted elsewhere.
   if (isCelo(chainId)) {
     if (address === nativeOnChain(chainId).wrapped.address) {
-      return 'https://raw.githubusercontent.com/ubeswap/default-token-list/master/assets/asset_CELO.png'
+      return 'https://celo-org.github.io/celo-token-list/assets/celo_alternative_logo.png'
```

Comment:
Why is this the _alternative_ logo?

Reply:
The alternative logo (yellow and white) is used for erc-20 transfers and is consistent with other applications on Celo. The [green and yellow](https://celo-org.github.io/celo-token-list/assets/celo_logo.svg) icon normally represents the network itself. 

For UI consistency; Celo on the celo token list (and others alike) use the alternative logo to represent $CELO. If $CELO is selected from the token list, the same logo should be displayed on the swap widget.